src/main/docs/guide/mongo.adoc

```diff
@@ -0,0 +1,10 @@
+Micronaut Data Mongo supports most of the things that are possible to do with JPA and JDBC/R2DBC implementations, including:
+
+- Repositories with compile-time generated filtering, aggregation and projection queries
+- Entities relations and cascading
+- Transactions
+- Joining relations
+- JPA Criteria API
+- Attribute converters
+
+In the first version of Micronaut Data Mongo only synchronous Mongo driver is supported.
```

Comment:
Hmm I think this is probably fine for M1 but we need to support reactive

Reply:
I have removed the reactive support to make the PR smaller, everything is ready for it to be implemented.